packages/autoloader/src/autoload.php

```diff
@@ -86,6 +86,7 @@ function autoloader( $class_name ) {
 						'Jetpack_Sync_Main',
 						'Automattic\Jetpack\Constants',
 						'Automattic\Jetpack\Tracking',
+						'Automattic\Jetpack\Tracking\Jetpack_Usage',
```

Comment:
Hey, we were supposed to make this list smaller 😄

Reply:
😅